src/pages/RedeemCode.tsx

```
@@ -0,0 +1,109 @@
+                alignItems: "center",
+                display: "flex",
+                backgroundImage: `url(${backgroundH})`,
+                backgroundSize: "cover",
+            }}
+        >
+            <div className="overlay" />
+            <h1 className="title">Godbless</h1>
+            <div
+                className="form-container"
+                onMouseEnter={(e) =>
+                    (e.currentTarget.style.transform = "scale(1.05)")
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)